govc/library/update.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package library
+
+import (
+	"context"
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/library"
+)
+
+type update struct {
+	*flags.ClientFlag
+
+	library.Library
+}
+
+func init() {
+	cli.Register("library.update", &update{})
+}
+
+func (cmd *update) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
+	cmd.ClientFlag.Register(ctx, f)
+
+	f.StringVar(&cmd.Name, "n", "", "Library name")
```

Comment:
```suggestion
	f.StringVar(&cmd.Library.Name, "n", "", "Library name")
```

Reply:
nit: consistency...instead of my suggestion, feel free to use **one** of both, embedded or named field.

Reply:
Question: are we good with these two parameters for now or do we need to support more field update operations?

Reply:
I've had this patch sitting around for a while, but iirc, the API (and UI) didn't support updating the other fields.